request_test.go

```diff
@@ -77,7 +84,7 @@ func TestRequestMarshalJSON(t *testing.T) {
 					Raw:  "raw-content",
 				},
 			},
-			"{\"url\":{\"raw\":\"http://www.google.fr\"},\"method\":\"POST\",\"body\":{\"mode\":\"raw\",\"raw\":\"raw-content\"}}",
+			"{\"url\":{\"raw\":\"http://www.google.fr\"},\"method\":\"POST\",\"body\":{\"mode\":\"raw\",\"raw\":\"raw-content\"qq}}",
```

Comment:
You have a typo here :
```suggestion
			"{\"url\":{\"raw\":\"http://www.google.fr\"},\"method\":\"POST\",\"body\":{\"mode\":\"raw\",\"raw\":\"raw-content\"}}",
```

Reply:
done.